appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
+  skip_symbols: false
+  artifact: core 
+  on:
+      appveyor_repo_tag: true
```

Comment:
The last thing I need to get going here is writing the build version number back into appveyor from the tag. Attempts so far have not been successful.